### PR TITLE
chore(charts): add configurable flags and dependencies to index.md

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

This PR adds the configurable flags and dependencies information to the index.md

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -37,6 +37,19 @@ helm install <your-relase-name> openebs-cstor/cstor --create-namespace
...
+## Dependencies
+
+By default this chart installs additional, dependent charts:
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://openebs.github.io/node-disk-manager | openebs-ndm | 1.0.2 |
+
+
+To disable the dependency during installation, set `openebsNDM.enabled` to `false`.
```
Comment (prateekpandey14) [inline index.md:49]: @shubham14bajpai  lets add the full command how to use set command for reference

File: index.md [hunk 2]
```diff
@@ -63,4 +76,132 @@ kubectl apply -f https://raw.githubusercontent.com/openebs/cstor-operators/maste
...
+| imagePullSecrets | string | `nil` | Image registry pull secrets |
+| openebsNDM.enabled | bool | `true` | Enable OpenEBS NDM dependency |
+| rbac.create | bool | `true` | Enable RBAC |
+| rbac.pspEnabled | bool | `false` | Enable PodSecurityPolicy |
+| release.version | string | `"2.5.0"` | Openebs CStor release version |
+| serviceAccount.annotations | object | `{}` | Service Account annotations |
+| serviceAccount.csiController.create | bool | `true` | Enable CSI Controller ServiceAccount |
+| serviceAccount.csiController.name | string | `"openebs-cstor-csi-controller-sa"` | CSI Controller ServiceAccount name |
+| serviceAccount.csiNode.create | bool | `true` | Enable CSI Node ServiceAccount |
+| serviceAccount.csiNode.name | string | `"openebs-cstor-csi-node-sa"` | CSI Node ServiceAccount name |
```
Comment (prateekpandey14) [inline index.md:207]: Here after the table we can show example , how to use the --set command with helm command.

File: index.md (hunk 1 above)
Comment (kmova) [reply index.md:49]: something to the effect of `helm install ... --set openebsNDM.enabled=false`

File: index.md (hunk 2 above)
Comment (kmova) [reply index.md:207]: Simliar to what we do in openebs/chart like:

---

Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.

Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,

```bash
helm install --name openebs-cstor -f values.yaml openebs-cstor/cstor 
```

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**